settings.gradle

```diff
@@ -6,7 +6,6 @@ include ":junit"
 include ":utils"
 include ":utils:reflector"
 include ":pluginapi"
-include ":plugins:accessibility-deprecated"
```

Comment:
nit: move this into a new PR

Reply:
I'll make a dedicated PR for that

Reply:
Change reverted in https://github.com/robolectric/robolectric/compare/d17a796d38a3b9e7055fd66c6bcca4a1b0082ffb..a8384cb033cae4c2e1ed54cd863ed799e5b03b03
I'll open a dedicated PR soon.